pkg/asset/manifests/azure/types.go

```diff
@@ -0,0 +1,96 @@
+package azure
+
+//authConfig is part of the CloudProviderConfig as defined in https://github.com/kubernetes/kubernetes/blob/v1.13.5/pkg/cloudprovider/providers/azure/auth/azure_auth.go#L32
+type authConfig struct {
```

Comment:
do we want to keep the un-used fields like `AAD*` ?

Reply:
This is what gets marshaled into the cloud provider config.
It's all the field available in the cloud provider, as defined by the provider itself (see link), so I suggest we keep these structs as is.

Reply:
:+1: